apps/teams-test-app/src/components/RememberScrollToggle.tsx

```diff
@@ -0,0 +1,54 @@
+import throttle from 'lodash.throttle';
+import React, { useEffect, useState } from 'react';
+
+const RememberScrollToggle = () => {
+  // Initialize the toggle state from sessionStorage
+  const [isScrollFeatureActive, setIsScrollFeatureActive] = useState(
+    () => JSON.parse(sessionStorage.getItem('isScrollFeatureActive') || '{}') ?? true,
+  );
+
+  useEffect(() => {
+    // Save the toggle state to sessionStorage whenever it changes
+    sessionStorage.setItem('isScrollFeatureActive', JSON.stringify(isScrollFeatureActive));
+
+    console.log('scrolling');
+
+    if (!isScrollFeatureActive) {
+      return;
+    }
+
+    const handleScroll = throttle(() => {
+      const scrollPosition = window.scrollY;
+      sessionStorage.setItem('iframeScrollPosition', JSON.stringify(scrollPosition));
+      console.log('scrolling');
+    }, 2000);
```

Comment:
What's your thoughts on changing the 2000 here to `const THROTTLE_TIME = 2000`?

Reply:
Agreed, anytime we need to use specific numbers like this it's always my preference to put them in something like a named const. Makes it much easier to find and change in the future, as well as document what the const is for, why it has that value, etc. Good to avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) whenever possible!